api/services/response/index.js

```diff
@@ -0,0 +1,14 @@
+const { statusMessagesHelper: statusMessages } = require('../../helpers');
+
+class BaseResponse {
+  getBody({ status = 200, data = {}, message = '' }) {
+    return {
+      status,
+      statusMessage: statusMessages[status],
+      message,
+      data,
+    };
```

Comment:
What is the logical sense of "message" field? Maybe it's better to remove it and keep just 3 fields (status, statusMessage, data)?

Reply:
Look,  statusMessage is a short message and it depends on file in the system. Field Message is a long message which contains additional information about result of this event. Such a way I think.

Reply:
DOne